client/src/components/Jobs/JobsStatus/JobsStatus.js

```diff
@@ -74,14 +74,14 @@ function JobsStatus({ status, fancy, jobId }) {
   let interactiveSessionLink;
 
   /* Check if job is running AND has interactive session */
-  if (status === 'RUNNING') {
-    const jobConcluded = [
-      'CLEANING_UP',
-      'ARCHIVING',
-      'FINISHED',
-      'STOPPED',
-      'FAILED'
-    ];
+  const jobConcluded = [
```

Comment:
maybe add a comment here?  just in case someone in the future sees this and says "man, it would be clearer if we just did `if (status === "RUNNING"` but is unaware of why you are doing it this way.